packages/legend-application-repl/src/stores/dataCube/DataCubeEngine.ts

```diff
@@ -84,8 +85,8 @@ export const DEFAULT_GRID_CLIENT_PURGE_CLOSED_ROW_NODES = false;
 export const DEFAULT_GRID_CLIENT_SUPPRESS_LARGE_DATASET_WARNING = false;
 
 export class DataCubeEngine {
-  readonly repl: REPLStore;
-  readonly application: LegendREPLApplicationStore;
+  readonly dataCube: DataCubeState;
```

Comment:
Don't do this, the point of the engine is it can be used to share across multiple `DataCubeState` so really what we need here is 


```suggestion
  readonly store: DataCubeStore;
```